diff --git a/node_modules/next-auth/providers/spotify.js b/node_modules/next-auth/providers/spotify.js
index 89989d0..45a6a3f 100644
--- a/node_modules/next-auth/providers/spotify.js
+++ b/node_modules/next-auth/providers/spotify.js
@@ -10,7 +10,7 @@ function Spotify(options) {
     id: "spotify",
     name: "Spotify",
     type: "oauth",
-    authorization: "https://accounts.spotify.com/authorize?scope=user-read-email",
+    authorization: "https://accounts.spotify.com/authorize?scope=user-read-email%20user-read-recently-played%20user-read-currently-playing%20playlist-read-private%20playlist-modify-private",
     token: "https://accounts.spotify.com/api/token",
     userinfo: "https://api.spotify.com/v1/me",
 
